thoth/common/workflows.py

```diff
@@ -166,6 +167,8 @@ def __deserialize(cls, body: Dict[str, str], *, validate: bool) -> "Workflow":
 class WorkflowManager:
     """Argo Workflow manager."""
 
+    LIMIT_WAIT_TIME = 2.5  # time to wait between limit checking of argo api
```

Comment:
Let's have it configurable via env var so we can adjust this in deployment without shipping a new component version.